packages/es-components/src/components/containers/tabPanels/Tab.js

```diff
@@ -33,8 +35,6 @@ const TabButton = styled.button`
     border-bottom-color: ${props =>
       props.selected ? ' transparent' : `${props.theme.colors.gray4}`};
     box-shadow: none;
-    color: ${props =>
-      props.selected ? props.theme.colors.black : props.theme.colors.primary};
```

Comment:
I think this is a breaking change. Older implementations will lose this style and they will have to provide new styling anywhere it's used.

If you want it backward-compatible a default style ought to be provided in the case the downstream implementer does not.

Reply:
The color here is already defined on line 14 and 15.  Is it necessary to restate the color specifically for a desktop design when it's the same color?  I just took it out because I thought it was redundant.  Maybe I'm misunderstanding something?

Reply:
Nope, that was my mistake! I overlooked those lines because they were collapsed in the code preview.